corehq/apps/app_manager/templates/app_manager/partials/modules/case_search_property.html

```diff
@@ -140,34 +140,38 @@ <h4 class="modal-title">
                   </div>
                 </div>
                 </fieldset>
-                {% if js_options.has_geocoder_privs %}
+                {% if js_options.has_geocoder_privs or js_options.exclude_from_search_enabled %}
                     <fieldset data-bind="visible: appearanceFinal() == 'fixture' || appearance() == ''">
```

Comment:
Just wanted to confirm this is logic that you want to apply to the new option. This means it'll only show for geocoder and text fields, right?

Reply:
My thinking was to only show the Advanced Options section if at least one of the two, `has_geocoder_privs` or `exclude_from_search_enabled` were true, and then populate the Advanced Options with one or both depending on the additional `if` checks below. I could be misunderstanding your question. Does that make sense?

Reply:
I think we're talking about slightly different things - and I wrote "geocoder" but should have written "lookup table," oops.

I meant to comment on this logic: `visible: appearanceFinal() == 'fixture' || appearance() == ''` which will hide the whole "Advanced Options" section if the field isn't a lookup table or text question (since the pre-existing advanced options are only relevant to search fields that are potential geocoder receiver fields for address/city/state/etc.)

That means your new option won't be available for barcode, date range, or geocoder broadcast fields. Is that intentional?